### PR TITLE
Testimony recording system fixes and additions

### PR DESCRIPTION
**Pull request by Lernos**
Added a few things to the testimony recording system. A handy quality-of-life OOC command for immediately clearing the testimony list for the area as well as a minor fix for amend_statement function and a new insert_statement one which allows players to insert a statement anywhere inside their testimony in the true spirit of "amending" it the way it was done in the games.

### CODE REVIEW

**Review by in1tiate — APPROVED** [review]
Looks good to me. Very useful additions.

File: server/area_manager.py [hunk 1]
```diff
@@ -608,6 +608,31 @@ def amend_statement(self, index: int, message: list) -> bool:
...
+                tempsti = [''] * (len(self.statements) + 1);
+                while i < index + 1:
+                    tempsti[i] = self.statements[i]
+                    i += 1
+                tempsti[index + 1] = message
+                i += 1
+                while i < len(tempsti):
+                    tempsti[i] = self.statements[i-1]
+                    i += 1
+                self.statements = tempsti
```
Comment (oldmud0) [inline server/area_manager.py]: [`list.insert`](https://docs.python.org/3/tutorial/datastructures.html)

**Review by oldmud0 — COMMENTED** [review]
i see big brain code

**Comment by Lernos** [comment]
Yurushite kudasai, sensei!

**Review by oldmud0 — APPROVED** [review]

**Pull request merged**